src/request/sign-transaction/SignTransaction.css

```diff
@@ -149,6 +149,10 @@
 }
 
 #confirm-transaction .data-section {
-    margin: 1rem 3rem;
+    margin: 1rem 0 1rem 3rem;
+    padding-right: 3rem;
     text-align: center;
+    max-width: 100%;
+    -webkit-mask: linear-gradient(90deg, white, white calc(100% - 3rem), rgba(255, 255, 255, 0));
+    mask: linear-gradient(90deg, white, white calc(100% - 3rem), rgba(255, 255, 255, 0));
```

Comment:
Is the overflow hidden? I don't think the data should ever be overflowing. The user has to see what he's signing.

Reply:
Yes, that's arguable. The whole data is still copyable when selecting-all or selecting the data section.

So you'd rather see a horizontal scrollbar there than hidden data? I mean this only happens when the text doesn't have spaces or other breakable points. Which is arguably the case e.g. for a HEX string.